src/auro-carousel.js

```diff
@@ -329,15 +345,15 @@ export class AuroCarousel extends LitElement {
         aria-roledescription="carousel"
         class="${classMap(carouselClassMap)}"
         @scroll=${() => this.setScrollFlags(true)} >
-        <button @click=${() => this.handleClick(false)} class="button button--left">
-          ${this.generateIconHtml(chevronLeft.svg)}<span class="util_displayHiddenVisually">Scroll Left</span>
-        </button>
+        <${this.buttonTag} arialabel="chevron-left" role="button" iconOnly rounded variant="secondary" @click=${() => this.handleClick(false)} class="button button--left">
```

Comment:
I believe this is the reason for the accessibility error. By identifying this custom tag as `role="button"` you are essentially putting a `button` inside a `button`, which is illegal. 

Why was `role="button"` added to the DOM element?